src/modules/unsafe/evalInVm.js

```diff
@@ -3,6 +3,7 @@ const {badValue} = require(__dirname + '/../config');
 const defaultLogger = require(__dirname + '/../utils/logger');
 const getObjType = require(__dirname + '/../utils/getObjType');
 const createNewNode = require(__dirname + '/../utils/createNewNode');
+const hash = require(__dirname + '/../utils/generateScriptHash')
```

Comment:
Looks great! Just to keep the style consistent, please import the name of the function as the name of the file, i.e. `generateScriptHash`.

Reply:
Done and done!